src/core/surface/channel_create.c

```diff
@@ -195,6 +198,7 @@ grpc_channel *grpc_channel_create(const char *target,
 #define MAX_FILTERS 3
   const grpc_channel_filter *filters[MAX_FILTERS];
   int n = 0;
+  GRPC_TIMER_BEGIN(PTAG_OTHER_BASE + 2, NULL);
```

Comment:
Let's remove this before the commit: I think in any work we do for now, channel creation will be in the noise.